webapp/src/services/profilelist/profilelist.service.js

```diff
@@ -20,4 +21,18 @@ function getProfilesList(event_id){
                       ? error.response.data.message :
                          error.message }
         });
+}
+
+function getUserProfile(user_id){
+    return axios
+            .get(baseUrl+"/api/v1/userprofile?user_id="+user_id.toString().split(":")[1],{headers: authHeader()})
```

Comment:
@mthembbu Can you explain your reasoning here? Why do we need to make it a string and split it?

Reply:
my mistake was not creating a new variable as an int, then parse that int instead.